### PR TITLE
MWPW-135907 - sign pdf and rotate pdf to l2 test

### PR DESCRIPTION
**Pull request by draganatrajkovic**
Resolves: https://jira.corp.adobe.com/browse/MWPW-135907
Test ticket for Rotate pdf: https://jira.corp.adobe.com/browse/MWPW-137180
Test ticket for Sign pdf: https://jira.corp.adobe.com/browse/MWPW-137251

### CODE REVIEW

**Review by zagi25 — APPROVED** [review]

File: test/e2e/frictionless/features/regression/converter.feature [hunk 1]
```diff
@@ -82,4 +82,39 @@ Feature: Frictionless Converter Block
 
   Examples:
       | Verb     | File                |
-      | sign-pdf | test-files/test.pdf |
\ No newline at end of file
+      | sign-pdf | test-files/test.pdf |
+
+  @MWPW-135907 @regression @converter
```
Comment (JFernandezAdobe) [inline test/e2e/frictionless/features/regression/converter.feature:87]: Please replace the tag with @MWPW-137180.

File: test/e2e/frictionless/features/regression/converter.feature [hunk 2]
```diff
@@ -82,4 +82,39 @@ Feature: Frictionless Converter Block
...
+    Then I rotate left first uploaded file
+    Then I wait for 2 seconds
+    Then I save rotated files
+    Then I download the converted file
+
+    Examples:
+      | Verb       | Files                                    |
+      | rotate-pdf | test-files/test.pdf,test-files/test2.pdf |
+
+  @MWPW-135907 @regression @converter
```
Comment (JFernandezAdobe) [inline test/e2e/frictionless/features/regression/converter.feature:101]: Please log a Test ticket and use the Jira ID as tag here.

**Review by JFernandezAdobe — CHANGES_REQUESTED** [review]
Thanks @draganatrajkovic! Please note that we have other tests for L2 verbs that focus on event-driven content. See [eventwrapper.feature](https://github.com/adobecom/dc/blob/main/test/e2e/frictionless/features/regression/eventwrapper.feature).

I logged a separate ticket for it - [MWPW-137181](https://jira.corp.adobe.com/browse/MWPW-137181).

File: test/e2e/frictionless/features/regression/converter.feature (hunk 1 above)
Comment (draganatrajkovic) [reply test/e2e/frictionless/features/regression/converter.feature]: I fixed this, can you please check?

File: test/e2e/frictionless/features/regression/converter.feature (hunk 2 above)
Comment (draganatrajkovic) [reply test/e2e/frictionless/features/regression/converter.feature]: This is also resolved, please check

**Review by JFernandezAdobe — APPROVED** [review]

**Review by TsayAdobe — APPROVED** [review]

**Pull request merged**